### ISSUE TITLE
Trace.plot() broken on ObsPy 0.10.2 and Matplotlib >= 1.5.0

### ISSUE THREAD

**Issue by claudiodsf** [obspy/obspy#1225]
```python
from obspy import read
st = read()
st.plot()
```
produces:
```
/Users/claudio/obspy_venv/venv/lib/python2.7/site-packages/matplotlib/axes/_base.py:2787: UserWarning: Attempting to set identical left==right results
in singular transformations; automatically expanding.
left=733643.0, right=733643.0
  'left=%s, right=%s') % (left, right))
```

and the following figure:

![figure_1](https://cloud.githubusercontent.com/assets/1390993/12509527/6e385102-c103-11e5-8315-639b61cabdf4.png)

Current development version of ObsPy works fine with Matplotlib >= 1.5.0.

If you can confirm this, we should either push a hot fix for ObsPy 0.10.2 or finalize the new version as soon as possible, since people are starting upgrading their Matplotlib installation.

**Comment by megies** [comment]
I can confirm this issue..

**Comment by QuLogic** [comment]
This is fixed by #1123.

**Comment by claudiodsf** [comment]
Sorry, I just checked the open issues before posting this.

My point however is another: Matplotlib 1.5.0 has been released at the end of October, ObsPy 0.10.2 is not compatible with Matplotlib >= 1.5.0 in a fundamental aspect (waveform plotting). In my opinion, this issue alone would justify a "0.10.2.1" release, unless 0.10.3 is really imminent.

**Comment by megies** [comment]
> This is fixed by #1123.

Oops, should've checked out `releases` head to try.. sorry.

**Comment by megies** [comment]
> In my opinion, this issue alone would justify a "0.10.2.1" release, unless 0.10.3 is really imminent

I guess we kind of silently agreed on doing the 0.10.3 in parallel with the next major release, in order to save some time and work doing two releases in parallel.. but yeah, we should really try to get both out soon. :/
